<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>541</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="SoundEffectWidget" name="kickWidget" native="true"/>
     </item>
     <item row="0" column="1">
      <widget class="SoundEffectWidget" name="snareWidget" native="true"/>
     </item>
     <item row="1" column="0">
      <widget class="SoundEffectWidget" name="hihatWidget" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="SoundEffectWidget" name="crashWidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SoundEffectWidget</class>
   <extends>QWidget</extends>
   <header>soundeffectwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
